barretenberg/cpp/src/barretenberg/stdlib/protogalaxy_verifier/protogalaxy_recursive_verifier.test.cpp

```diff
@@ -14,6 +14,8 @@
 #include "barretenberg/ultra_honk/ultra_prover.hpp"
 #include "barretenberg/ultra_honk/ultra_verifier.hpp"
 
+auto& engine = bb::numeric::get_debug_randomness();
```

Comment:
I had to debug an issue with the relation parameters, and using this deterministic randomness made sure that the same parameters were used from run to run in my test.